tests/components/test_conversation.py

```diff
@@ -46,6 +50,9 @@ def test_register_before_setup(hass):
     """Test calling an intent from a conversation."""
     intents = async_mock_intent(hass, 'OrderBeer')
 
+    result = yield from component.async_setup(hass, {})
```

Comment:
Why does this need to be setup?

Reply:
__init__ for components doesn't explicitly get called so the intents aren't registered without it.

Reply:
but this is a test for OrderBeer intent, not one of the control devices intents?

Reply:
Arg, you are right of course, went a little nuts with the cut/paste. Sorry you are having to review this so much, I have been multitasking and not giving it as much undivided attention as I should.

Reply:
So can we remove this line?